runtime/protocol/http-client-engines/test-suite/common/test/aws/smithy/kotlin/runtime/http/test/AsyncStressTest.kt

```diff
@@ -22,6 +23,9 @@ class AsyncStressTest : AbstractEngineTest() {
     fun testConcurrentRequests() = testEngines {
         // https://github.com/awslabs/aws-sdk-kotlin/issues/170
         concurrency = 1_000
+        engineConfig {
+            maxConcurrency = 32u
```

Comment:
Your PR description says 16 but this is 32, which is correct?

Reply:
It was 16 I bumped it to 32 (hence the _closer_). We can move it to 16 or continue increasing it until it breaks. I didn't realize it was 16 until I looked it up after testing it.